### PR TITLE
Update VS extension version to 5.3 and some minor fixes

### PR DESCRIPTION
**Pull request by pratikone**
# PR checklist

Updates VS extension versions to 5.3

** Minor fix **
Refer https://github.com/MicrosoftDocs/winrt-related/issues/19
In packaged apps, the app manifest was missing the changes made here
https://github.com/microsoft/WindowsAppSDK/pull/3206/files#

**Which issue does this PR relate to?**

**Applies to the following platforms:**

- [X] WinUI
- [X] WPF
- [X] UWP

### CODE REVIEW

**Review by mikebattista — APPROVED** [review]

File: code/TemplateStudioForWinUICpp/Templates/Features/MSIX/Param_ProjectName (Package)/Package.appxmanifest [hunk 1]
```diff
@@ -12,7 +13,9 @@
     Publisher="CN=Param_UserName"
     Version="1.0.0.0" />
 
-  <Properties>
+   <mp:PhoneIdentity PhoneProductId="07ad151b-a614-4a13-aef0-5378d3f98e67" PhonePublisherId="00000000-0000-0000-0000-000000000000"/>
```
Comment (wjk) [inline code/TemplateStudioForWinUICpp/Templates/Features/MSIX/Param_ProjectName (Package)/Package.appxmanifest:16]: I don’t think we should put the same GUID in _all_ projects that are generated by this template. The Windows Store may give users grief if they update a package with a PhoneProductId that has already been used by another app.

File: code/TemplateStudioForWinUICs/Templates/Ft/MSIX/Package.appxmanifest [hunk 2]
```diff
@@ -13,7 +14,9 @@
     Name="07ad151b-a614-4a13-aef0-5378d3f98e67"
     Publisher="CN=Param_UserName"
     Version="1.0.0.0" />
-
+    
+  <mp:PhoneIdentity PhoneProductId="07ad151b-a614-4a13-aef0-5378d3f98e67" PhonePublisherId="00000000-0000-0000-0000-000000000000"/>
```
Comment (wjk) [inline code/TemplateStudioForWinUICs/Templates/Ft/MSIX/Package.appxmanifest:18]: Ditto.

**Review by wjk — CHANGES_REQUESTED** [review]

File: code/TemplateStudioForWinUICpp/Templates/Features/MSIX/Param_ProjectName (Package)/Package.appxmanifest (hunk 1 above)
Comment (mikebattista) [reply code/TemplateStudioForWinUICpp/Templates/Features/MSIX/Param_ProjectName (Package)/Package.appxmanifest:16]: These GUIDs get replaced during project generation.

Comment (Scottj1s) [reply code/TemplateStudioForWinUICpp/Templates/Features/MSIX/Param_ProjectName (Package)/Package.appxmanifest:16]: To remove any doubt/confusion, why not zero the guids out then?

Comment (mikebattista) [reply code/TemplateStudioForWinUICpp/Templates/Features/MSIX/Param_ProjectName (Package)/Package.appxmanifest:16]: template.json supports GUID replacements like https://github.com/microsoft/TemplateStudio/blob/6c3907aa00e08752a15b2394a01cc9d14222ede5/code/TemplateStudioForWinUICs/Templates/Ft/MSIX/.template.config/template.json#L30.

Each GUID listed gets replaced with a unique GUID. Given the templates were already following this pattern and this change just applied the replacement GUID to another element, I don't see a need to change, but you could 0 out the GUIDs and remove the autogeneration if you wanted.

**Pull request merged**